building/configlet/lint.md

```diff
@@ -418,3 +418,16 @@ The `config.json` file should have the following checks:
 - `%{snake_slug}`: the `snake_case` exercise slug (e.g. `bit_manipulation`)
 - `%{camel_slug}`: the `camelCase` exercise slug (e.g. `bitManipulation`)
 - `%{pascal_slug}`: the `PascalCase` exercise slug (e.g. `BitManipulation`)
+5. Unique version 4 UUID string: A string that satisfies all of these conditions:
+   - It only exists once in the track-level `config.json` file of a given Exercism track
+   - It does not exist in the track-level `config.json` file of any other Exercism track
+   - It does not exist in any `canonical-data.json` file in https://github.com/exercism/problem-specifications
```

Comment:
Should we add other places that a UUID can appear?

Or should we just add something like this?
> It does not exist anywhere else on Exercism

Reply:
I like the second option best.

Reply:
Done.